openmetadata-ui/src/main/resources/ui/src/components/NavBar/NavBar.tsx

```diff
@@ -128,6 +147,17 @@ const NavBar = ({
     });
   }, []);
 
+  const handleSupportClick = ({ key }: MenuInfo) => {
+    switch (key) {
+      case HELP_ITEMS_ENUM.WHATS_NEW:
+        setIsFeatureModalOpen(true);
+
+        break;
+      default:
+        break;
+    }
+  };
```

Comment:
if there is only one condition then, if condition will be more readable in my opinion